src/components/VTimeline/VTimelineItem.js

```diff
@@ -0,0 +1,183 @@
+// Components
+import VIcon from '../VIcon'
+
+// Mixins
+import Colorable from '../../mixins/colorable'
+
+export default {
+  name: 'v-timeline-item',
+
+  inject: [
+    'iconParent',
+    'iconSizeParent',
+    'noIconParent',
+    'lineColorParent',
+    'hoverParent',
+    'raisedParent',
+    'circleFillColorParent',
+    'circleOutlineColorParent',
+    'lineSizeParent',
+    'hideCircleOutlineParent'
+  ],
+
+  mixins: [Colorable],
+
+  props: {
+    raised: {
+      type: Boolean,
+      default: false
+    },
+    hover: {
+      type: Boolean,
+      default: false
+    },
+    icon: {
+      type: String
+    },
+    iconSize: {
+      type: [Number, String]
+    },
+    noIcon: {
+      type: Boolean,
+      default: false
+    },
+    lineColor: {
+      type: String
+    },
+    circleOutlineColor: {
+      type: String
+    },
+    circleFillColor: {
+      type: String
+    },
+    hideCircleOutline: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  computed: {
+    classes () {
+      return this.addBackgroundColorClassChecks({
+        'timeline__item--body': true,
+        'timeline--hover': this.hover || this.hoverParent,
+        'timeline--raised': this.raised || this.raisedParent
+      })
+    },
+    lineClasses () {
+      const lineColor = this.lineColor ? this.lineColor : this.lineColorParent
+      return this.addTextColorClassChecks({}, lineColor)
+    },
+    circleClasses () {
+      const circleFillColorClass = this.circleFillColor ? this.circleFillColor : this.circleFillColorParent
+      const classes = {
+        [circleFillColorClass]: true
+      }
+      const circleOutlineColor = this.circleOutlineColor
+        ? this.circleOutlineColor
+        : this.circleOutlineColorParent ? this.circleOutlineColorParent : this.lineColor ? this.lineColor : this.lineColorParent
```

Comment:
`this.circleOutlineColor || this.circleOutlineColorParent || this.lineColor || this.lineColorParent`